gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/store/ManagedDirectBufferAllocator.java

```diff
@@ -60,6 +62,13 @@
   public static final String DIRECT_STORE_OBJECT_OWNER =
       "SNAPPYDATA_DIRECT_STORE_OBJECTS";
 
+  public static final String CACHED_DATA_FRAME_RESULTOUTPUT_OWNER =
+          "CACHED_DATA_FRAME_RESULTOUTPUT";
+
+  public static final List<String> nonEvictingOwners = new ArrayList() {{
+    add("CACHED_DATA_FRAME_RESULTOUTPUT_OWNER");
+  }};
```

Comment:
Name is incorrect -- it should be the variable name not string.

Is this used somewhere? Didn't see.